W06H01/src/test/PinguFoodLogisticsTest.java

```diff
@@ -23,117 +23,131 @@
-    PrepareConsole();
-    
+    assertContains(baos, "Bisher konnten 17 Tiere mit einem Gesamtgewicht von 2289g nicht verwertet werden." + lineSeparator +
+      "Claudia und Karl-Heinz ist dadurch ein Profit von 33525PD entgangen.");
+    resetConsole();
+
     pfl1.acceptNewOrder(new TradeOrder());
     pfl1.acceptNewOrder(new WeightOrder(0));
     pfl1.acceptNewOrder(new AmountOrder(0, 0, 0));
     pfl1.clearOrderBook();
-    assertTrue(baos.toString().contains("Es können 3 Bestellungen abgearbeitet werden."));
-    assertTrue(baos.toString().contains("Die Bestellung(Einzeln) hat ein Gesamtgewicht von 2g und kostet 20PD."));
-    PrepareConsole();
+    assertContains(baos, "Es können 3 Bestellungen abgearbeitet werden.");
+    assertContains(baos, "Die Bestellung(Einzeln) hat ein Gesamtgewicht von 2g und kostet 20PD.");
+    resetConsole();
     pfl1.printWasteStatistics();
-    assertTrue(baos.toString().contains("Bisher konnten 17 Tiere mit einem Gesamtgewicht von 2289g nicht verwertet werden.\n" +
-            "Claudia und Karl-Heinz ist dadurch ein Profit von 33525PD entgangen."));
-    
+    assertContains(baos, "Bisher konnten 17 Tiere mit einem Gesamtgewicht von 2289g nicht verwertet werden." + lineSeparator +
+      "Claudia und Karl-Heinz ist dadurch ein Profit von 33525PD entgangen.");
+
     pfl1.acceptNewOrder(new TradeOrder());
     pfl1.acceptNewOrder(new WeightOrder(500));
     pfl1.acceptNewOrder(new AmountOrder(9, 7, 8));
     pfl1.clearOrderBook();
-    assertTrue(baos.toString().contains("Es können 3 Bestellungen abgearbeitet werden."));
-    assertTrue(baos.toString().contains("Die Bestellung(Einzeln) hat ein Gesamtgewicht von 10g und kostet 100PD."));
-    assertTrue(baos.toString().contains("Die Bestellung(Zielgewicht: 500g) hat ein Gesamtgewicht von 597g und kostet 7180PD."));
-    assertTrue(baos.toString().contains("Die Bestellung(Anzahl: [9,7,8]) hat ein Gesamtgewicht von 2029g und kostet 27340PD."));
-    PrepareConsole();
+    assertContains(baos, "Es können 3 Bestellungen abgearbeitet werden.");
+    assertContains(baos, "Die Bestellung(Einzeln) hat ein Gesamtgewicht von 10g und kostet 100PD.");
+    assertContains(baos, "Die Bestellung(Zielgewicht: 500g) hat ein Gesamtgewicht von 597g und kostet 7180PD.");
+    assertContains(baos, "Die Bestellung(Anzahl: [9,7,8]) hat ein Gesamtgewicht von 2029g und kostet 27340PD.");
+    resetConsole();
     pfl1.printWasteStatistics();
-    assertTrue(baos.toString().contains("Bisher konnten 32 Tiere mit einem Gesamtgewicht von 4452g nicht verwertet werden.\n" +
-            "Claudia und Karl-Heinz ist dadurch ein Profit von 65510PD entgangen."));
-    
+    assertContains(baos, "Bisher konnten 32 Tiere mit einem Gesamtgewicht von 4452g nicht verwertet werden." + lineSeparator +
+      "Claudia und Karl-Heinz ist dadurch ein Profit von 65510PD entgangen.");
+
+  }
+
+  private void assertContains(ByteArrayOutputStream baos, String needle) {
```

Comment:
I mainly use predefined functions of JUnit to easy other students the path of learning JUnit. But otherwise this improves the readability of the code.